tests/haproxy/master-failover-test.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# Ensure admin username and admin password are passed to script in Jenkinsfile.
+if  [[ -z $1 ]] || [[ -z $2 ]]
+then
+	echo "Usage $0 <admin username> <admin password>"
+	exit 1
+fi
+
+# Move to correct directory to run commands with relative paths
+cd /usr/share/ansible/openshift-deployment-ansible/tests/haproxy 
+
+# Logging into openshift in order to run oc commands later in script. 
+DOMAINSUFFIX=$(cat ../../group_vars/all.yml | grep domainSuffix | awk '{print $2}')
+
+oc login -u $1 -p $2 --server="https://ocp.$DOMAINSUFFIX:8443" --insecure-skip-tls-verify
+
+# Storing master node names (easiest way to do this and ensure future proof for node changes)
+master0name=$(cat ../../openshift-ansible-hosts | fgrep -C3 [masters] | tail -3 | head -1)
```

Comment:
use the following instead
masters="$(oc get nodes -o custom-columns=NAME:metadata.name --no-headers=true | grep master)"
master0name=$("${masters}" | grep -- -0\.)